src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/ThorchainSaversDeposit.tsx

```diff
@@ -152,7 +156,7 @@ export const ThorchainSaversDeposit: React.FC<YearnDepositProps> = ({
       [DefiStep.Info]: {
         label: translate('defi.steps.deposit.info.title'),
         description: translate('defi.steps.deposit.info.description', {
-          asset: underlyingAsset?.symbol ?? '',
+          asset: isRunePool ? thorchainAsset?.symbol : underlyingAsset?.symbol ?? '',
```

Comment:
preferably-blocking: update `underlyingAssetId` l.108 to be `opportunityData?.underlyingAssetId`

Reply:
Absolutely, this is way better! Fixed in [12e90a7](https://github.com/shapeshift/web/pull/7535/commits/12e90a7ee14c33fe20cc5f581ca0bc015cd1df33)